packages/muelu/test/unit_tests/BlockedRepartition.cpp

```diff
@@ -886,7 +886,7 @@ namespace MueLuTests {
   RCP<RepartitionHeuristicFactory> RepHeuFact = Teuchos::rcp(new RepartitionHeuristicFactory);
   RepHeuFact->SetFactory("A",AcFact);//MueLu::NoFactory::getRCP());
   RepHeuFact->SetParameter("repartition: start level",Teuchos::ParameterEntry(0));
-  RepHeuFact->SetParameter("repartition: min rows per proc",Teuchos::ParameterEntry(40));
+  RepHeuFact->SetParameter("repartition: min rows per proc",Teuchos::ParameterEntry(100));
```

Comment:
Why did this need to be changed?

Reply:
It didn't.  I just wanted to trigger the issue on 2 ranks rather than 4.

Reply:
For debugging purposes.